src/ContentType/Request/RequestHandlerInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of sensiolabs-de/storyblok-bundle.
+ *
+ * (c) SensioLabs Deutschland <info@sensiolabs.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Storyblok\Bundle\ContentType\Request;
+
+use Storyblok\Api\Response\StoryResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\HttpExceptionInterface;
+
+interface RequestHandlerInterface
+{
+    /**
+     * @throws HttpExceptionInterface
+     */
+    public function handle(Request $request, string $slug): Response|StoryResponse;
```

Comment:
@OskarStark i am not quite happy with this. Do you have any better idea? I don't like to have two return types here:

- `Symfony\Component\HttpFoundation\Request`
- `Storyblok\Api\Response\StoryResponse`

Reply:
Hmm, not really. Couldn't we always return the final response?
Or other.... when do we return a Response and when do we return a StoryResponse?

Reply:
any idea @smnandre ?

Reply:
I dropped this.